static/app/views/explore/components/schemaHintsList.tsx

```diff
@@ -211,10 +212,10 @@ function SchemaHintsList({
             () => (
               <SchemaHintsDrawer
                 hints={filterTagsSorted}
-                exploreQuery={exploreQuery}
+                exploreQuery={query}
                 tableColumns={tableColumns}
                 setPageParams={setPageParams}
-                source={source}
+                searchBarDispatch={dispatch}
```

Comment:
Instead of passing this down, we can just use the hook in the schema hints drawer component to get it